config.go

```diff
@@ -338,3 +338,31 @@ func nodejs(c *Config) error {
 
 	return nil
 }
+
+// python config.
+func python(c *Config) {
+	if c.Proxy.Command == "" {
+		c.Proxy.Command = "python app.py"
+	}
+
+	// Only add build & clean hooks if a requiremnts.txt exists
+	if !util.Exists("requirements.txt") {
+		return
+	}
+
+	// Set PYTHONPATH env
+	if c.Environment == nil {
+		c.Environment = config.Environment{}
+	}
+	c.Environment["PYTHONPATH"] = ".pypath/"
+
+	// Copy libraries into .pypath/
+	if c.Hooks.Build == "" {
+		c.Hooks.Build = `mkdir -p .pypath/ && pip install -r requirements.txt -t .pypath/`
```

Comment:
What about packages that need to compile? If I'm not mistaken, unless the machine running `up` has the same OS as Lambda, this will result in failure.